openapi/components/schemas/Quotes/Quote.yaml

```
@@ -0,0 +1,157 @@
+type: object
+required:
+  - websiteId
+  - customerId
+  - items
+properties:
+  id:
+    readOnly: true
+    description: ID of the quote.
+    type: string
+    maxLength: 50
+    example: qt_0YV7DES3WPC5J8JD8QTVNZBZNZ
+  orderType:
+    description: Specifies the type of quote. A quote can be a subscription or a one-time purchase.
+    type: string
+    readOnly: true
+    enum:
+      - subscription-order
+      - one-time-order
+  status:
+    description: Status of quote.
+    type: string
+    readOnly: true
+    enum:
+      - draft
+      - issued
+      - accepted
+      - rejected
+      - cancelled
+      - expired
+    x-enumDescriptions:
+      draft: Quote can be edited. This quote cannot be sent to a customer.
+      issued: Quote cannot be edited. This quote can be sent to a customer.
+      accepted: Customer accepted the quote and created an order.
+      rejected: Customer rejected the quote.
+      cancelled: Organization cancelled the quote.
+      expired: Expired before customer or organization interaction.
+  websiteId:
+    $ref: ../Websites/WebsiteId.yaml
+  customerId:
+    $ref: ../CustomerId.yaml
+  orderId:
+    description: ID of the order.
+    readOnly: true
+    nullable: true
+    type: string
+    maxLength: 50
+    example: ord_0YV7DES3WPC5J8JD8QTVNZBZNZ
+  items:
+    type: array
+    minItems: 1
+    items:
+      $ref: ./QuoteItem.yaml
+  deliveryAddress:
+    description: Delivery address of the order.
+    nullable: true
+    type: object
+    allOf:
+      - $ref: ../Contact/ContactObject.yaml
+  billingAddress:
+    description: Billing address of the order.
+    nullable: true
+    type: object
+    allOf:
+      - $ref: ../Contact/ContactObject.yaml
+  invoicePreview:
+    $ref: ./QuoteInvoicePreview.yaml
+  paymentTerms:
+    description: Payment terms for the customer which are displayed on the quote.
+    type: string
+  expirationTime:
+    description: |-
+      Date and time when the quote expires. The default expiration time is one month from the time the quote is issued.
+      This field can be `null` in `draft` state.
```

suggestion: suggestion for readability 
```suggestion
      In a `draft` state, this field may be `null`.
```